contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -692,6 +692,7 @@ class Resource extends mix(APIResource, IndexedDBResource) {
         // Only fetch new updates if we've finished syncing the changes table
         db[CHANGES_TABLE].where('table')
           .equals(this.tableName)
+          .filter(c => !c.synced)
```

Comment:
The previous `if` block appears to be the cause of https://github.com/learningequality/studio/issues/3699, since the backend may not have created the folder yet. Perhaps we should consolidate the logic for the two blocks here, using this change detection?

Reply:
Or just add a special case for `parent` filtering and detecting it as a new folder. Not really in scope of this PR, but this change made me wonder if it would fix the other bug (it doesn't)